packages/block-template-react/tsconfig.json

```diff
@@ -7,7 +7,8 @@
     "lib": ["dom", "dom.iterable", "esnext"],
     "module": "commonjs",
     "noEmit": true,
-    "target": "es6"
+    "target": "es6",
+    "resolveJsonModule": true
```

Comment:
had to add this in-order to import the block's package.json. See https://github.com/blockprotocol/blockprotocol/pull/557/files#r959809732